xml/System.Security.Cryptography/HashAlgorithmName.xml

```diff
@@ -289,7 +289,15 @@ Due to collision problems with MD5 and SHA1, Microsoft recommends a security mod
       <Docs>
         <summary>Gets the underlying string representation of the algorithm name.</summary>
         <value>The string representation of the algorithm name, or <see langword="default" /> if no hash algorithm is available.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+May be `null` or empty to indicate that no hash algorithm is applicable.
```

Comment:
Already in the return value description.

Reply:
Removing.